### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by kideny**
update chinese translate

### CODE REVIEW

**Comment by sergeyklay** [comment]
@dreamsxin Could you please take a look

File: zh/index.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ Phalcon 不只是为了卓越的性能, 我们的目标是让它更加健壮，
 目录
 -----
 
-Installation
+安装
 ^^^^^^^^^^^^
```
Comment (dreamsxin) [inline zh/index.rst]: 将^个数设置为汉字个数2倍

**Comment by dreamsxin** [comment]
That's ok.

**Comment by kideny** [comment]
@dreamsxin translate is update

**Comment by dreamsxin** [comment]
@kideny 下次你可以自己生成一次，测试
```shell
sudo apt-get install sphinx-common
# or
sudo apt install python-pip
git clone https://github.com/dreamsxin/sphinx.git
cd sphinx
pip install . -i http://pypi.douban.com/simple --trusted-host pypi.douban.com

cd zh
make html
```

**Comment by kideny** [comment]
@dreamsxin i get it

File: zh/_theme/phalcon/layout.html [hunk 2]
```diff
@@ -153,7 +153,7 @@
       };
     </script>
 
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.min.js"></script>
+    <script src="http://apps.bdimg.com/libs/jquery/1.8.1/jquery.min.js"></script>
```
Comment (sergeyklay) [inline zh/_theme/phalcon/layout.html]: Could you please explain this?

Comment (kideny) [inline zh/_theme/phalcon/layout.html]: googleapis.com  is very snowly in China . the  new url is very fast in China, baidu.com support.

Comment (kideny) [inline zh/_theme/phalcon/layout.html]: this modify is very importent for Chinese Phalcon fans ,

**Comment by SidRoberts** [comment]
I'll merge this in several hours. Don't worry about the merge conflict - I'll deal with it.

**Comment by SidRoberts** [comment]
Added in https://github.com/phalcon/docs/commit/4f8032991b105dc374939d8c8edea029f98544fd.

**Pull request closed without merging**